client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -65,7 +66,7 @@ class PurchaseMeta extends Component {
 	renderPrice() {
 		const { purchase, translate } = this.props;
 		const { priceText, currencyCode, productSlug } = purchase;
-		const plan = getPlan( productSlug );
+		const plan = getPlan( productSlug ) || getProductFromSlug( productSlug );
```

Comment:
I'm worried that this pattern will lead to further bugs when `getPlan` is called in a context where a product could be passed in and `getProductFromSlug` isn't also checked. However I think combining products and plans isn't going to be trivial so I'm fine shipping this as-is and doing that later.

Reply:
I agree! I've made a different ticket to track this implementation though, as it's something we need to deal with eventually. But outside of the scope of those, imo.